### ISSUE TITLE
How to duplicate a row?

### ISSUE THREAD

**Issue by Josko91** [czastack/RszTool#17]
I'm using Rzs Tool 0.3.0-net472 and I can only Copy and Paste but I can't duplicate any of the rows in the file?

**Comment by czastack** [comment]
You can duplicate GameObject or array item. And copy and paste instance or instance field.
GameObject is in pfb and scn file.

**Comment by Josko91** [comment]
So I can't dublicate anything in user files?

I want to add a new instance in playerlasersightcontreolleruserdata.user.2 file
chainsaw.PlayerLaserSightControllerUserData.Weapon.Settings[6]  this one is for the SG-09 R Handgun, 

I saw the mod "Laser for every handgun" have more instances, I tried copy and paste but it would only overwrite an existing instance, not make a new one. I have no idea how others have added new instances. Maybe I need to use another tool for that?
